src/deploy-cromwell-on-azure/scripts/allowed-vm-sizes

```diff
@@ -0,0 +1,12 @@
+﻿# Use this file to constraint the Azure VM sizes used for Cromwell task execution on Azure Batch
+# If no VM sizes are listed here, then all supported VM sizes will be eligible
+# VM size names should be placed one per line
+# Lines starting with # (comments) are ignored
+# Comments in the same line as the VM size are not allowed
+# For the full list of VM sizes and their features, see file "supported-vm-sizes" in this container
+# After making changes to this file, restart your Cromwell on Azure host VM
+
+# Examples:
+# Standard_D1
```

Comment:
We should consider providing a better default, based on our known experience.  For example - removing all 1-core and "A" machines, and any v1/v2 D-series.

Reply:
I removed all Basic and 1 CPU machines from the list. File allowed-vm-sizes is owned by the user, so not listing all the VMs in there because new VMs will then not be picked up unless user adds them.